src/feed/tests/views/test_funding_status_priority_sorting.py

```diff
@@ -0,0 +1,215 @@
+        return [item["content_object"]["id"] for item in response.data["results"]]
+
+    def test_proposals_tab_open_before_closed_with_amount_raised(self):
+        """Proposals tab: OPEN fundraises appear before CLOSED when ordering by amount_raised"""
+        open_post = self._create_fundraise_post("Open Proposal", Fundraise.OPEN, 10)
+        closed_post = self._create_fundraise_post(
+            "Closed Proposal", Fundraise.COMPLETED, -5
+        )
+
+        response = self.client.get(
+            reverse("funding_feed-list")
+            + "?ordering=amount_raised"
+        )
+
+        post_ids = self._get_post_ids(response)
+        self.assertIn(open_post.id, post_ids, "Open post should be in results")
+        self.assertIn(closed_post.id, post_ids, "Closed post should be in results")
+        self.assertLess(post_ids.index(open_post.id), post_ids.index(closed_post.id))
+
+    def test_proposals_tab_open_before_closed_with_hot_score(self):
+        """Proposals tab: OPEN before CLOSED when ordering by hot_score"""
+        open_post = self._create_fundraise_post("Open Hot", Fundraise.OPEN, 10)
+        closed_post = self._create_fundraise_post("Closed Hot", Fundraise.COMPLETED, -5)
+
+        response = self.client.get(
+            reverse("funding_feed-list") + "?ordering=hot_score"
+        )
+
+        post_ids = self._get_post_ids(response)
+        self.assertIn(open_post.id, post_ids)
+        self.assertIn(closed_post.id, post_ids)
+        self.assertLess(post_ids.index(open_post.id), post_ids.index(closed_post.id))
+
+    def test_proposals_tab_open_before_closed_default_ordering(self):
+        """Proposals tab: OPEN before CLOSED with default (newest) ordering"""
+        open_recent = self._create_fundraise_post(
+            "Open Recent", Fundraise.OPEN, 5
+        )
+        closed_old = self._create_fundraise_post(
+            "Closed Old", Fundraise.COMPLETED, -30
+        )
+
+        response = self.client.get(
+            reverse("funding_feed-list")
+        )
+
+        post_ids = self._get_post_ids(response)
+        self.assertIn(open_recent.id, post_ids)
+        self.assertIn(closed_old.id, post_ids)
+        self.assertLess(
+            post_ids.index(open_recent.id), post_ids.index(closed_old.id)
+        )
+
+    def test_request_for_proposals_open_before_closed(self):
+        """Request for Proposals tab: OPEN grants appear before CLOSED grants"""
+        open_grant = self._create_grant_post("Open RFP", Grant.OPEN, 10)
+        closed_grant = self._create_grant_post("Closed RFP", Grant.CLOSED, -5)
+
+        response = self.client.get(reverse("grant_feed-list"))
+
+        post_ids = self._get_post_ids(response)
+        self.assertIn(open_grant.id, post_ids)
+        self.assertIn(closed_grant.id, post_ids)
+        self.assertLess(
+            post_ids.index(open_grant.id), post_ids.index(closed_grant.id)
+        )
+
+    def test_all_tab_open_before_closed(self):
```

Comment:
The concept of "tab" doesn't mean much for the backend. It was kinda confusing when reading the code.

Name like `test_open_proposals_show_before_closed` is easier to read